routers/routes/routes.go

```diff
@@ -404,6 +404,16 @@ func RegisterRoutes(m *macaron.Macaron) {
 			Post(bindIgnErr(auth.CreateRepoForm{}), repo.ForkPost)
 	}, reqSignIn)
 
+	// ***** Release Attachment Download without Sigin
+	m.Get("/:username/:reponame/releases/download/:vTag/:fileName", ignSignIn, context.RepoAssignment(), repo.RedirectDownload)
```

Comment:
This is can not be accepted to allow file download without authorization. This is security flaw